example.py

```diff
@@ -1,9 +1,11 @@
 import csv
 import sys
+import pdb
```

Comment:
Do you want to leave pdb (and pdb.set_trace() below) in the code or was that for your own debugging?